examples/finetuning/dpo_tic_tac_toe/src/dpo_tic_tac_toe/move_search_strategy.py

```
@@ -0,0 +1,160 @@
+    def supported_pipeline_types(self) -> list[PipelineTypeEnum]:
+        """Support agent execution and custom pipeline types."""
+        return [PipelineTypeEnum.AGENT_EXECUTION, PipelineTypeEnum.CUSTOM]
+
+    def stage_type(self) -> StageTypeEnum:
+        """This is a search strategy."""
+        return StageTypeEnum.SEARCH
+
+    async def ainvoke(
+        self,
+        items: list[TTCItem],
+        original_prompt: str | None = None,
```

Ruff (0.14.8)

Unused method argument: `original_prompt`

(ARG002)